scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -118,9 +118,15 @@ function Initialize-IoTEdge {
         return
     }
 
+    if ((Test-MobyNeedsToBeMoved) -or (Test-LegacyInstaller)) {
+        Write-HostRed
+        Write-HostRed 'IoT Edge is installed in an invalid location. To reinstall, run "Uninstall-IoTEdge" first.'
```

Comment:
Maybe

>To reinstall, first remove the existing installation using "Uninstall-IoTEdge".

so that it doesn't sound like the command to reinstall is called "Uninstall-".

Same for the others.

Reply:
Amended.

Reply:
Missed a location, fixing that now.